workbench-core/base/src/interfaces/filterRequest.ts

```diff
@@ -3,8 +3,9 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { QueryParameterFilter } from './queryParameterFilter';
+import { QueryNumberParamFilter } from './queryNumberParamFilter';
+import { QueryStringParamFilter } from './queryStringParamFilter';
 
 export interface FilterRequest {
-  [key: string]: QueryParameterFilter<unknown> | undefined;
+  [key: string]: QueryStringParamFilter | QueryNumberParamFilter;
```

Comment:
Record<string, QueryStringParamFilter | QueryNumberParamFilter>

Reply:
![Screen Shot 2022-11-16 at 5 26 28 PM](https://user-images.githubusercontent.com/3661906/202307962-57586e30-c222-48cc-b175-0d68e3a4d396.png)

 :( TS lint was not happy using `Record`

Reply:
We might have to change `FilterRequest` to be a type to use Record.

Reply:
sorry we don't want to change the property, i think we want to do `export interface FilterRequest extends Record<string, QueryStringParamFilter | QueryNumberParamFilter> {}`. if that doesn't work though we can just keep it as is, i might be misunderstanding what is happening here.

Reply:
That worked. Updating the code.